packages/grid/x-data-grid/src/models/api/gridParamsApi.ts

```diff
@@ -12,6 +13,24 @@ export interface GridParamsApi {
    * @returns {v} The cell value.
    */
   getCellValue: <V extends any = any>(id: GridRowId, field: string) => V;
+  /**
+   * Gets the value of a row
+   * @template V
+   * @param {GridRowModel} row The row model.
+   * @param {GridColDef} colDef The column definition.
+   * @returns {v} The cell value.
+   * @ignore - do not document
+   */
+  getRowValue: <V extends any = any>(row: GridRowModel, colDef: GridColDef) => V;
+  /**
+   * Gets the formatted value of a row
```

Comment:
```suggestion
   * Gets the cell formatted value
   * Use it instead of `getCellParams` for better performance if you only need the formatted value.
```